taylorseager/podcast-player
# set remove podcast ternary

## Description
Remove button will only show when viewing podcasts within a playlist.

## Related Issue
#36 

## Motivation and Context
This change will allow users to only see the remove button when they are viewing the podcasts within a playlist. The user will not see the remove button when viewing all podcasts.

## How Can This Be Tested?
Login ->
click "Podcasts" -> confirm there is no remove button on podcast cards
click "Playlists" -> click "View" -> confirm there is a remove button on the podcast cards

## Screenshots (if appropriate):

## Types of changes
